### ISSUE TITLE
Check on contact requests test: potential false positive results

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/desktop-qa-automation#450]
https://ci.status.im/job/status-desktop/job/systems/job/linux/job/x86_64/job/tests-e2e-new/167/allure/#suites/64a2e13a8c3e80b0638402823e89ed26/26ccda08d31b72fb/

I logged a bug today that contact requests are dead https://github.com/status-im/status-desktop/issues/13246
Community invite test failed because of that (which is expected). However, tests that are checking contact requests, are all green.

That is suspicious. We need to make sure the tests are working and check why they are passing when they must fail (no contact request received).

<img width="1840" alt="Screenshot 2024-01-19 at 20 44 19" src="https://github.com/status-im/desktop-qa-automation/assets/82375995/637a03e6-74da-41b3-8059-40e0af1fb0a6">

**Comment by Valentina1133** [comment]
Hello @anastasiyaig . I checked tests and see that contact request was accepted in fact. Even in join community test - it failed not at the step of accepting contact request but at the step of accepting community invitation. So if step with accepting contact request was false positive, the test would fail earlier (user would not be able to send community invitation to the user which didn't accept contact request). 

But it is interesting indeed why it passes in tests if there is a bug

**Comment by Valentina1133** [comment]
I investigated more but I still don'e understand. I checked on https://ci.status.im/job/status-desktop/job/nightly/155/, it's a build with bug. No issue - contact requests sent and received.

Also I checked all the steps - it's just impossible that they pass with not receiving request. There are many steps which would fail in this case. 

So I don't know what is it but it's not the issue with tests for 100%.
